shell/packages/sandstorm-db/migrations.js

```diff
@@ -496,6 +496,36 @@ function smtpPortShouldBeNumber() {
   }
 }
 
+function consolidateOrgSettings() {
+  const orgGoogleDomain = Settings.findOne({ _id: "organizationGoogle" });
+  const orgEmailDomain = Settings.findOne({ _id: "organizationEmail" });
+  const orgLdap = Settings.findOne({ _id: "organizationLdap" });
+  const orgSaml = Settings.findOne({ _id: "organizationSaml" });
```

Comment:
If all the above returned `undefined` then should we skip the rest of the function?